### ISSUE TITLE
terminator opens a new tab in HOME directory instead of current one

### ISSUE THREAD

**Issue by ynikitenko** [gnome-terminator/terminator#429]
**Describe the bug**
When I open terminator with `terminator -g /dev/null`, it opens two tabs from my config (I don't understand why - I thought that is for an empty config. Anyway, when I press a new tab (Ctrl+Shift+t), it opens a new tab in home directory (even though I was in, say `mydir`). However, if I press Enter in a tab and then call a new tab by that shortcut, it opens in `mydir`.

I noticed this in another way. I open terminator with Ctlr+Alt+t. It opens two tabs with a directory from my config. They work fine, but if I open terminator again with that command and open a new tab, then that new tab will be again the home directory!

**Expected behavior**
Should open same directory as the current tab.

**Desktop (please complete the following information):**
 - Arch Linux.
 - Version 2.1.1.

**Additional context**
I reported some problems with my config earlier, the config is [here](https://github.com/gnome-terminator/terminator/issues/414#issuecomment-813062726) if needed.
I found similar (but other) issues https://github.com/gnome-terminator/terminator/issues/262, https://github.com/gnome-terminator/terminator/issues/99, both closed.

**Comment by mattrose** [comment]
Umm, the fact that it's reading your config even though you told it not to says to me that you're probably attaching to an existing terminator instance with your config loaded.

what happens when you try `terminator -u -g /dev/null`

**Comment by ynikitenko** [comment]
I see. Now I try this command when there is no opened terminator window.
It opens a new terminator window in my directory, but with the directory specified in my config. There is only one tab though (from the config it should be 2). Tabs are opened in that same directory, so it seems the problem is only when my config is somehow in effect (when I call terminator with Ctrl+Alt+t).
BTW, when I opened terminator today, every time I pressed Ctrl, it shrinked the font. Not sure if I shall create a new issue or start a config from scratch.
